hatasmota/mqtt.py

```diff
@@ -3,9 +3,10 @@
 from __future__ import annotations
 
 import asyncio
+import logging
+import requests
```

Comment:
The `hatasmota` library is mostly written with asyncio instead of threads for parallelism. Could you try to use `aiohttp` instead of `requests` for the firmware check?